src/prefect/cli/worker.py

```diff
@@ -222,6 +240,35 @@ async def _check_work_pool_paused(work_pool_name: str) -> bool:
         return False
 
 
+async def _check_work_queues_paused(
+    work_pool_name: str, work_queues: Optional[List[str]]
+) -> bool:
+    """
+    Check if all work queues in the work pool are paused. If work queues are specified,
+    only those work queues are checked.
+
+    Args:
+        - work_pool_name (str): the name of the work pool to check
+        - work_queues (Optional[List[str]]): the names of the work queues to check
+
+    Returns:
+        - bool: True if work queues are paused, False otherwise
+    """
+    try:
+        work_queues_filter = (
+            WorkQueueFilter(name=WorkQueueFilterName(any_=work_queues))
+            if work_queues
+            else []
```

Comment:
hmm should this be `None`?
```suggestion
            else None
```

Reply:
Ah yeah, good call, I was thinking the `else []` was for `any_` but yeah, it's for `work_queues_filter` so that makes sense.